App/Dependencies/AppDependencies+DebugMenu.swift

```diff
@@ -107,14 +107,6 @@
           title: "🎮 [Status] Simulate Recover Confirmed",
           dispatchable: Logic.CovidStatus.UpdateStatusWithEvent(event: .userEvent(.recoverConfirmed))
         ),
```

Comment:
<!--
  0 failure: 
  1 warning:  swiftlint (0.39.2...
  
  
  DangerID: danger-id-Danger;
  File: App/Dependencies/AppDependencies+DebugMenu.swift;
  Line: 109;
-->

- :warning: swiftlint (0.39.2): Collection literals should not have trailing commas. (trailing_comma)